packages/training-link/src/TrainingLink.tsx

```diff
@@ -12,13 +12,24 @@ export type TrainingLinkProps = {
 };
 
 const TrainingLink = ({ name, link }: TrainingLinkProps): JSX.Element => (
-  <span style={shouldProbablyBeInUIKit} className="ml-auto">
-    Need Help?{' '}
-    <a href={link} style={linkStyles} className="link" target="_blank" rel="noopener noreferrer">
-      Watch a demo <span className="sr-only">(opens in a new tab)</span>
-    </a>{' '}
-    for {name}
-  </span>
+  <>
+    {name ? (
+      <span style={shouldProbablyBeInUIKit} className="ml-auto">
+        Need Help?{' '}
+        <a href={link} style={linkStyles} className="link" target="_blank" rel="noopener noreferrer">
+          Watch a demo <span className="sr-only">(opens in a new tab)</span>
+        </a>{' '}
+        for {name}
+      </span>
+    ) : (
+      <span style={shouldProbablyBeInUIKit} className="ml-auto">
+        Need Help?{' '}
+        <a href={link} style={linkStyles} className="link" target="_blank" rel="noopener noreferrer">
+          Learn More
```

Comment:
Does this not open in a new tab/window as well?

Reply:
@LauRoxx  My PO said no need for the note about opens in new window

Reply:
It is there for accessibility and needs to remain. please add it for the learn more link as well.

Reply:
sure. I will add it